packages/blockchain/README.md

```diff
@@ -24,29 +24,24 @@ The following is an example to iterate through an existing Geth DB (needs `level
 
 This module performs write operations. Making a backup of your data before trying it is recommended. Otherwise, you can end up with a compromised DB state.
 
-```javascript
-const level = require('level')
-const Blockchain = require('@ethereumjs/blockchain').default
-const utils = require('ethereumjs-util')
+```typescript
+import Blockchain from '@ethereumjs/blockchain'
+import { bufferToInt } from 'ethereumjs-util'
+import level from 'level'
 
 const gethDbPath = './chaindata' // Add your own path here. It will get modified, see remarks.
+
 const db = level(gethDbPath)
+const blockchain = new Blockchain({ db })
 
-new Blockchain({ db: db }).iterator(
-  'i',
-  (block, reorg, cb) => {
-    const blockNumber = utils.bufferToInt(block.header.number)
-    const blockHash = block.hash().toString('hex')
-    console.log(`BLOCK ${blockNumber}: ${blockHash}`)
-    cb()
-  },
-  (err) => console.log(err || 'Done.'),
-)
+blockchain.iterator('i', (block) => {
```

Comment:
Is this intended to leave the `await` here?

Reply:
Yes the await would be useful for awaiting until the iterator is finished before moving onto the next line of code but since there is none I thought it’d be okay (and then I’d have to wrap it in an inner async func since you can’t await from the top level). I didn’t actually test it but I think it’ll work like that! Please correct me if I’m wrong :)

Reply:
I guess another option could be to add a `.then(() => { console.log('Done.') }).catch(console.error)`